base/markdown/render/plain.jl

```diff
@@ -20,9 +20,12 @@ function plain{l}(io::IO, header::Header{l})
 end
 
 function plain(io::IO, code::Code)
-    println(io, "```", code.language)
+    # If the code includes a fenced block this will break parsing,
+    # so it must be enclosed by a longer ````-sequence.
+    n = mapreduce(length, max, matchall(r"`+", code.code)) + 1
```

Comment:
Yeah, you're right. Though you actually still need the + 1 in there. Have updated.